astroquery/mast/tests/test_mast_remote.py

```diff
@@ -150,10 +150,6 @@ def test_mast_service_request_async(self):
         assert isinstance(responses, list)
 
     def test_mast_service_request(self):
-
-        # clear columns config
-        Mast._column_configs = dict()
```

Comment:
it looks like this and the other assignments to dict() were done to reset the class attribute `_column_configs`. Are we sure we don't need this anymore?

Reply:
I'm not 100% sure what purpose the lines serve, but removing them has no impact on the test results. From what I can tell, `_column_configs` seems to be a sort of caching variable to avoid having to make multiple calls for the `columnsConfig` entry for a service. I don't see any apparent reason that this should be reset for certain tests, since I don't imagine that we expect the configuration for a service to change all that often. What do you think?

Reply:
i'm fine with removing from the tests